packages/oauth/src/form/form-service.ts

```diff
@@ -176,7 +176,7 @@ export class FormService implements OAuthProtoService {
   }
 
   async isLoggedIn(): Promise<boolean> {
-    return await this.login
+    return this.login
```

Comment:
Don't forget to remove `async` above.

Reply:
Darn! That sneaked in there!

Reply:
Removed `async` but Promise must remain to conform with interface since the OS OAuthProtoService is definitely asynchronous in the method.